eventhub/Module.md

```diff
@@ -127,17 +105,17 @@ You can specify the event hub path and the event data as parameters of the send
 This operation will return a ballerina error if the operation failed.
 
 
-## Entity Management in an Azure Event Hub 
+### Entity Management in an Azure Event Hub 
```

Comment:
Is this a new Quick start?

Reply:
Merged into one now. Fixed in df7c75b . Thank you.